### PR TITLE
WIP Feature/help button

### LINKED ISSUE #72: Need Help Modal [Booking Sys Func]
Need to create modal plus entry points in settings for the 3-4 help tips.

In settings, it should just be a couple of textareas.

### PR DESCRIPTION
**Pull request by APM246**
## Proposed Changes
  - Work in progress help button that pops up a modal
  - Uses react-modal library from https://github.com/reactjs/react-modal

image

### CODE REVIEW

**Comment by APM246** [comment]
As for the settings page, I was thinking of doing the same modal idea. Could you clarify what you mean by "entrypoints for settings" in the Issue page.

**Comment by zollf** [comment]
it's so that they can update the help modals. So in settings, there be a textarea called 'Step 1 Help' and whatever goes in there will appear in Step 1 help button. For now, this is fine as someone still needs to hook up the backend and frontend for settings and other values.

File: app/resources/static/documentation/docs.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const carpark = 'Please select the carpark you would like to make a booking in';
```
Comment (zollf) [inline app/resources/static/documentation/docs.ts]: files in static are accessible by url. We should not have .ts files in here. Please put the file within some other folder in the `frontend` folder.

**Comment by APM246** [comment]
Oh right I see. I thought you were referring to having help guides in the admin panel itself.

**Review by zollf — CHANGES_REQUESTED** [review]

**Comment by APM246** [comment]
> it's so that they can update the help modals. So in settings, there be a textarea called 'Step 1 Help' and whatever goes in there will appear in Step 1 help button. For now, this is fine as someone still needs to hook up the backend and frontend for settings and other values.

I'd be happy to do that. I'll create an issue for it if that's all good with you

**Comment by zollf** [comment]
Nws

**Review by zollf — APPROVED** [review]

**Pull request merged**